pom.xml

```diff
@@ -293,6 +293,12 @@
                     <configuration>
                         <argLine>${argLine} -Duser.timezone=UTC -Dddf.version=${ddf.version}
                         </argLine>
+                        <includes>
+                            <include>**/*Test.java</include>
+                            <include>**/Test*.java</include>
+                            <include>**/*TestCase.java</include>
+                            <include>**/*Spec.class</include>
```

Comment:
Yep, bug related to how surefire handles groovy files. Need to use `*Spec.class`, can't use `*Spec.groovy`.